final_prep/README.md

```diff
@@ -23,52 +23,64 @@ Some things to consider as you refactor include...
-- [ ] I have refactored my `section3` exercises to the best of my ability
+- [ X] I have refactored my `section1` exercises to the best of my ability
+- [ X] I have refactored my `section2` exercises to the best of my ability
+- [ X] I have refactored my `section3` exercises to the best of my ability
 
 ## Gearing Up for Gear Ups
 
-Gear Up is a part of our curriculum where we talk about topics such as privilege and power, and ethical issues in the tech industry. 
+Gear Up is a part of our curriculum where we talk about topics such as privilege and power, and ethical issues in the tech industry.
+
+
+
+- [ X] [Complete this assignment](https://github.com/turingschool/gear-up/blob/main/m0/Intro.To.GearUp.md)
+
+  - [ X] Add a link for your gist here: [My Gear Up Intro Gist](https://gist.github.com/bensjsilverstein/9159c547cc1a14381e3e5f287b62c567)
 
-- [ ] [Complete this assignment](https://github.com/turingschool/gear-up/blob/main/m0/Intro.To.GearUp.md)
-  - [ ] Add a link for your gist here: 
 
 ## Time Management Prep
 
 In Mod 0 you've learned about different techniques for managing your time at Turing. Please create a calendar for your **first 3 weeks of Mod 1**. Feel free to make your calendar fit your style, but we suggest that your calendar should include the following:
 
-- [ ] Standard Mod 1 class schedule (find the Mod 1 calendar [on the Back End curriculum site](https://backend.turing.edu/)). The detailed classes held on your actual start date may not yet be posted; you can look back at the previous inning to get a feel for what Weeks 1-3 will be like, but know it's not guaranteed to be exactly the same. Also, as you already know, class is held from 9-12 and 1-4MT, Monday-Friday.
-- [ ] Study/Project work time (usually about 25 hours/week outside of class time)
-- [ ] Personal, Health + Wellness
+- [ X] Standard Mod 1 class schedule (find the Mod 1 calendar [on the Back End curriculum site](https://backend.turing.edu/)). The detailed classes held on your actual start date may not yet be posted; you can look back at the previous inning to get a feel for what Weeks 1-3 will be like, but know it's not guaranteed to be exactly the same. Also, as you already know, class is held from 9-12 and 1-4MT, Monday-Friday.
+- [ X] Study/Project work time (usually about 25 hours/week outside of class time)
+- [ X] Personal, Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+
+- [Week 1](https://user-images.githubusercontent.com/102967531/164868073-e0358a99-acd6-4637-a935-7bfcb59227bb.png)
+
+- [Week 2](https://user-images.githubusercontent.com/102967531/164868032-d13e6e2e-ec80-4c95-8ec8-f7b1c46adb8b.png)
+
+- [Week 3](https://user-images.githubusercontent.com/102967531/164868145-c9d6c9bb-9229-411a-bcc1-0ecf81e11c9d.png)
```

Comment:
For this, I see about 1.5 hours/day -> 7.5/week of outside-of-class time dedicated to study/practice - most students are in the 20-30 range. What are the times of day you are going to get that work in, be available to collaborate with groupmates, etc?